src/main/java/com/revature/model/InterviewInterviewer.java

```diff
@@ -0,0 +1,83 @@
+package com.revature.model;
+import java.util.Objects;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "interview_interviewer")
+public class InterviewInterviewer {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name="interview_interviewer_id")
+	private int id;
+	private String text;
+	@ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+	@JoinColumn(name="fk_interview")
+	private Interview interview;
+	private String interviewer;
```

Comment:
Perhaps add a comment specifying that this would be the interviewer's username.

Reply:
Changes are made as per suggestion